lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/persistence/DBOMessageToUserBackup.java

```diff
@@ -0,0 +1,231 @@
+package org.sagebionetworks.repo.model.dbo.persistence;
```

Comment:
Please open a Jira issue to remove DBOMessageToUserBackup in the subsequent stack.

Reply:
Tracking that task here https://sagebionetworks.jira.com/browse/PLFM-4735